telluric/vectors.py

```diff
@@ -274,11 +274,16 @@ def __init__(self, shape, crs=DEFAULT_CRS):
             Geometry.
         crs : ~rasterio.crs.CRS (optional)
             Coordinate Reference System, default to :py:data:`telluric.constants.DEFAULT_CRS`.
+        safe: bool, optional
+            Check method arguments validity (only CRS so far) if False,
+            default to True
 
         """
         self._shape = shape  # type: shapely.geometry.base.BaseGeometry
         self._crs = CRS(crs)
-        assert self._crs.is_valid
+
+        if not safe:
+            assert self._crs.is_valid
```

Comment:
Perhaps I would use this opportunity to raise a proper error instead of a blank `AssertionError`?

Reply:
Sorry, I'm saying nonsense: this already raises a proper exception.